package.json

```diff
@@ -48,6 +48,7 @@
     "babel-preset-stage-1": "^6.3.13",
     "exenv": "^1.2.0",
     "inline-style-prefixer": "^0.6.2",
+    "mobile-detect": "^1.3.1",
```

Comment:
Preventing default sounds dangerous.  `mobile-detect` does user-agent sniffing which is not what you want anyway.  Just do:

```javascript
const isMobile = typeof window.ontouchstart !== "undefined";
```